jupyter_client/channels.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, context=None, session=None, address=None):
         atexit.register(self._notice_exit)
 
         self._running = False
-        self._pause = True
+        self._pause = False
```

Comment:
@minrk While I believe the fix is correct here, it's a little counterintuitive to have ``_running`` and ``_pause`` to both be False. Perhaps comment?